compatibility/sandbox-migration/util.bzl

```diff
@@ -53,4 +54,4 @@ def migration_test(name, versions, tags, quick_tags, **kwargs):
         ] + [dep for ver in oracle_versions(versions) for dep in runfiles(ver)],
         args = ["--oracle"] + oracle_versions(versions),
         **kwargs
-    ) if len(oracle_versions(versions)) > 1 else None
+    ) if len(oracle_versions(versions)) > 1 and is_linux else None
```

Comment:
this means you cannot run it locally for testing which doesn’t seem great. How about we slap a `manual` tag on it on macos so you can still run it locally but it’s not run by default and thereby also not run by CI?

Reply:
Good point, changed to a manual tag.